tsconfig.json

```diff
@@ -7,24 +7,22 @@
     "lib": ["es2020", "esnext.bigint", "es2020.string", "es2020.symbol.wellknown"],
     // Export declaration files '.d.ts' to /lib
     "declaration": true,
-    "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
 
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
-    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    "alwaysStrict": true /* Parse in strict mode and emit "use strict" for each source file. */,
     "strictNullChecks": true /* Enable strict null checks. */,
-    // "strictFunctionTypes": true,           /* Enable strict checking of function types. */
-    // "strictBindCallApply": true,           /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
-    // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
-    // "noImplicitThis": true,                /* Raise error on 'this' expressions with an implied 'any' type. */
-    // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
-
-    /* Additional Checks */
-    // "noUnusedLocals": true,                /* Report errors on unused locals. */
+    "strictFunctionTypes": true /* Enable strict checking of function types. */,
+    "strictBindCallApply": true /* Enable strict 'bind', 'call', and 'apply' methods on functions. */,
+    "strictPropertyInitialization": true /* Enable strict checking of property initialization in classes. */,
+    "noImplicitAny": true /* Raise error on expressions and declarations with an implied 'any' type. */,
+    "noImplicitThis": true /* Raise error on 'this' expressions with an implied 'any' type. */,
+    "noImplicitReturns": true /* Report error when not all code paths in function return a value. */,
+    "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,
```

Comment:
this one is a bit annoying

Reply:
Since we use a lot of enums + switch I think it can bring more value than pain. You can always use a default if necessary

Reply:
it prevents grouping like:
```
Enum.ONE:
Enum.TWO: doSomething(); break;
Enum.THREE:
Enum.FOUR: doSomethingElse(); break;
default: throw new Error();
```
I don't really see what value it brings, never seen project with that flag enabled tbh haha If you enable it using if-else makes more sense than switch-case